### PR TITLE
Fix behavior field crashing the app in InlineEdition

### PR DESCRIPTION
**Pull request by ClementPasteau**

### CODE REVIEW

File: newIDE/app/src/EventsSheet/ParameterFields/RelationalOperatorField.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const mapTypeToOperators: { [string]: Array<string> } = {
 };
 
 export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
-  function GlobalVariableField(props: ParameterFieldProps, ref) {
+  function RationalOperatorField(props: ParameterFieldProps, ref) {
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/ParameterFields/RelationalOperatorField.js:32]:
```suggestion
  function RelationalOperatorField(props: ParameterFieldProps, ref) {
```

File: newIDE/app/src/EventsSheet/ParameterFields/ObjectField.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ import InAppTutorialContext from '../../InAppTutorial/InAppTutorialContext';
 
 export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
   function ObjectField(props: ParameterFieldProps, ref) {
+    const { currentlyRunningInAppTutorial } = React.useContext(
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/ParameterFields/ObjectField.js:17]: Useful or just for style?

File: newIDE/app/src/EventsSheet/ParameterFields/BehaviorField.js [hunk 3]
```diff
@@ -4,127 +4,138 @@ import * as React from 'react';
...
-
-  constructor(props: ParameterFieldProps) {
-    super(props);
-
-    const { parameterMetadata } = this.props;
-    this._description = parameterMetadata
+export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
+  function BehaviorField(props: ParameterFieldProps, ref) {
+    const field = React.useRef<?SemiControlledAutoCompleteInterface>(null);
+    const focus: FieldFocusFunction = options => {
```
Comment (AlexandreSi) [inline newIDE/app/src/EventsSheet/ParameterFields/BehaviorField.js:18]: use callback?

**Review by AlexandreSi — APPROVED** [review]
LGTM

**Review by 4ian — APPROVED** [review]

**Pull request merged**